### PR TITLE
Add an endpoint for search results that returns dummy data

### PR DESCRIPTION
**Pull request by jguertin**
Create the basic structure for `GET /searches/{id}/results` and add a reachable endpoint.  For the moment it contains dummy data, but will have it query the db in future commits

### CODE REVIEW

File: ppr-api/src/endpoints/search.py [hunk 1]
```diff
@@ -43,3 +46,24 @@ def read_search(search_id: int, search_repository: repository.search_repository.
     if search_db_row is None:
         raise fastapi.HTTPException(status_code=404, detail="Search record not found")
     return search_db_row
+
+
+@router.get("/searches/{search_id}/results", response_model=typing.List[schemas.search.SearchResult],
```
Comment (WalterMoar) [inline ppr-api/src/endpoints/search.py]: Double quotes.

**Review by WalterMoar — CHANGES_REQUESTED** [review]
Just some double quotes, otherwise looks good.

File: ppr-api/src/endpoints/search.py (hunk 1 above)
Comment (bryan-gilbert) [reply ppr-api/src/endpoints/search.py]: Does the lint checker catch this?

Comment (WalterMoar) [reply ppr-api/src/endpoints/search.py]: No, but Justin identified some packages that might work. We'll have the same problem in the front end, with the added bonus that in a SFC we'll need doubles in the template and css, and singles in the script 😮

**Review by bryan-gilbert — APPROVED** [review]

File: ppr-api/src/endpoints/search.py (hunk 1 above)
Comment (jguertin) [reply ppr-api/src/endpoints/search.py]: Corrected with the rebase

**Review by WalterMoar — APPROVED** [review]
Woohoo.

**Pull request merged**